phase/inject_arm_fixes.go

```diff
@@ -0,0 +1,81 @@
+package phase
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/k0sproject/k0sctl/config"
+	"github.com/k0sproject/k0sctl/config/cluster"
+)
+
+// InjectARMFixes implements a phase which fixes arm quirks (must run before uploadfiles)
+type InjectARMFixes struct {
+	GenericPhase
+
+	hosts cluster.Hosts
+}
+
+// Title for the phase
+func (p *InjectARMFixes) Title() string {
+	return "Fix ARM quirks"
+}
+
+// Prepare the phase
+func (p *InjectARMFixes) Prepare(config *config.Cluster) error {
+	p.Config = config
+
+	hosts := p.Config.Spec.Hosts.Filter(func(h *cluster.Host) bool {
```

Comment:
I suppose this is only needed for controllers

```suggestion
	hosts := p.Config.Spec.Hosts.Filter(func(h *cluster.Host) bool {
	if h.Role == "worker" {
		return false
	}
```